### PR TITLE
Add encoding parameter

### PR DESCRIPTION
**Pull request by Its-Alex**
I need to set encoding using this library, is it possible to add this option ?

### CODE REVIEW

File: reconnecting_ftp/__init__.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def __init__(self,
                  port: int,
                  user: str,
                  password: str,
+                 encoding: str = 'utf-8',
```
Comment (mristin) [inline reconnecting_ftp/__init__.py]: Could you please move the encoding to the end of the argument list to avoid breaking the existing interface?

**Review by mristin — CHANGES_REQUESTED** [review]
Thanks a lot! Please see a minor comment.

File: reconnecting_ftp/__init__.py (hunk 1 above)
Comment (Its-Alex) [reply reconnecting_ftp/__init__.py]: @mristin Sure! It's done 🙂

**Comment by mristin** [comment]
@Its-Alex thanks again! The new version should be on PyPI now.

P.S. if you haven't done already, please star the project on GitHub to give it some visibility.

**Pull request merged**